server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMailboxesMethodTest.java

```diff
@@ -1441,4 +1441,39 @@ public void setMailboxesShouldUpdateMailboxNameWhenRenamingAChildMailbox() {
             .body(ARGUMENTS + ".list", hasSize(1))
             .body(ARGUMENTS + ".list[0].name", equalTo("mySecondBox"));
     }
+
```

Comment:
I think the question remains if the system mailbox has not been already created.